components.go

```diff
@@ -630,92 +651,100 @@ func (cal *Calendar) Todos() []*VTodo {
 	return r
 }
 
-func (c *VTodo) SetCompletedAt(t time.Time, params ...PropertyParameter) {
-	c.SetProperty(ComponentPropertyCompleted, t.UTC().Format(icalTimestampFormatUtc), params...)
+func (todo *VTodo) SetCompletedAt(t time.Time, params ...PropertyParameter) {
+	todo.SetProperty(ComponentPropertyCompleted, t.UTC().Format(icalTimestampFormatUtc), params...)
 }
 
-func (c *VTodo) SetAllDayCompletedAt(t time.Time, params ...PropertyParameter) {
+func (todo *VTodo) SetAllDayCompletedAt(t time.Time, params ...PropertyParameter) {
 	params = append(params, WithValue(string(ValueDataTypeDate)))
-	c.SetProperty(ComponentPropertyCompleted, t.Format(icalDateFormatLocal), params...)
+	todo.SetProperty(ComponentPropertyCompleted, t.Format(icalDateFormatLocal), params...)
 }
 
-func (c *VTodo) SetDueAt(t time.Time, params ...PropertyParameter) {
-	c.SetProperty(ComponentPropertyDue, t.UTC().Format(icalTimestampFormatUtc), params...)
+func (todo *VTodo) SetDueAt(t time.Time, params ...PropertyParameter) {
+	todo.SetProperty(ComponentPropertyDue, t.UTC().Format(icalTimestampFormatUtc), params...)
 }
 
-func (c *VTodo) SetAllDayDueAt(t time.Time, params ...PropertyParameter) {
+func (todo *VTodo) SetAllDayDueAt(t time.Time, params ...PropertyParameter) {
 	params = append(params, WithValue(string(ValueDataTypeDate)))
-	c.SetProperty(ComponentPropertyDue, t.Format(icalDateFormatLocal), params...)
+	todo.SetProperty(ComponentPropertyDue, t.Format(icalDateFormatLocal), params...)
 }
 
-func (c *VTodo) SetPercentComplete(p int, params ...PropertyParameter) {
-	c.SetProperty(ComponentPropertyPercentComplete, strconv.Itoa(p), params...)
+func (todo *VTodo) SetPercentComplete(p int, params ...PropertyParameter) {
+	todo.SetProperty(ComponentPropertyPercentComplete, strconv.Itoa(p), params...)
 }
 
-func (c *VTodo) SetGeo(lat interface{}, lng interface{}, params ...PropertyParameter) {
-	c.setGeo(lat, lng, params...)
+func (todo *VTodo) SetGeo(lat interface{}, lng interface{}, params ...PropertyParameter) {
+	todo.setGeo(lat, lng, params...)
 }
 
-func (c *VTodo) SetPriority(p int, params ...PropertyParameter) {
-	c.setPriority(p, params...)
+func (todo *VTodo) SetPriority(p int, params ...PropertyParameter) {
+	todo.setPriority(p, params...)
 }
 
-func (c *VTodo) SetResources(r string, params ...PropertyParameter) {
-	c.setResources(r, params...)
+func (todo *VTodo) SetResources(r string, params ...PropertyParameter) {
+	todo.setResources(r, params...)
```

Comment:
Thats only refactoring, but improved readability is nice 👍

Reply:
It's work by @quite in PRs like: https://github.com/arran4/golang-ical/pull/105 He didn't miss any but there was perhaps a communication problem that caused a couple to get reverted. So I applied it here too.